File: Python/palindrome_number.py
a = int(input("Enter a no:"))
sum = 0
n = a
while(n>0):
    d = n % 10
    sum=sum*10 + d
    n = n // 10
if(a == sum):
    print("Palindrome detected")
else:
    print("Palindrome not detected")

File: C/PalindromeNumber.c
#include <stdio.h>
int main() {
    int num, rev=0, rem, num1;
    printf("Enter an integer: ");
    scanf("%d", &num);
    num1 = num;
    
    while (num != 0) {
        rem = num % 10;
        rev = rev * 10 + rem;
        num /= 10;
    }

    if (num1 == rev)
        printf("%d is a palindrome.", num1);
    else
        printf("%d is not a palindrome.", num1);

    return 0;
}
